src/config.rs

```diff
@@ -114,6 +114,9 @@ pub struct Config {
     /// Build the project only when a file got saved and not on file change. Default: false
     pub build_on_save: bool,
     pub use_crate_blacklist: bool,
+    /// Cargo target dir. If set overrides the default one.
+    #[serde(skip_deserializing, skip_serializing)]
+    pub target_dir: Option<PathBuf>,
```

Comment:
I think this is a good approach, however I wonder if it can be improved and the fact that it's an override could be better indicated. Not sure if possible and how to do that myself, though.